.circleci/config.yml

```diff
@@ -345,45 +290,9 @@ workflows:
           name: UI Tests (iPad)
           <<: *ipad_test_device
           requires: [ "Optional Tests" ]
-  Installable Build:
-    when:
-      and:
-        - not: << pipeline.parameters.beta_build >>
-        - not: << pipeline.parameters.release_build >>
-    jobs:
-      - Approve WordPress:
-          type: approval
-          filters:
-            branches:
-              ignore: /pull\/[0-9]+/
-      - WordPress Installable Build:
-          requires: [Approve WordPress]
-          filters:
-            branches:
-              ignore: /pull\/[0-9]+/
-      - Approve Jetpack:
-          type: approval
-          filters:
-            branches:
-              ignore: /pull\/[0-9]+/
-      - Jetpack Installable Build:
-          requires: [Approve Jetpack]
-          filters:
-            branches:
-              ignore: /pull\/[0-9]+/
 
   Release Build:
     when:
       or: [ << pipeline.parameters.beta_build >>, << pipeline.parameters.release_build >> ]
     jobs:
       - Release Build
-  Jetpack Nightly:
-    triggers:
-       - schedule:
-           cron: "0 0 * * *"
-           filters:
-             branches:
-               only:
-                 - develop
-    jobs:
-      - Jetpack Installable Build
```

Comment:
I checked the Buildkite schedules tab and there is no equivalent for this.

Has it been deemed unnecessary?

Reply:
I'll let Jeremy reply but my guess is, since we'll now create Installable Builds for all PRs without waiting for confirmation (i.e. we won't have the "Hold" job anymore), this indeed kinda makes those Nightly Builds irrelevant anymore?

Reply:
Yep, this is the case – we can also trivially re-add it with a separate pipeline file, but I'd like to see if it's actually needed :)